install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,602 @@ $ oc label node --all logging-infra-fluentd=true <1>
+is the unit name for a session.
+
+| `systemd.t.SYSTEMD_USER_UNIT`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#_SYSTEMD_CGROUP=[`systemd.t.SYSTEMD_USER_UNIT`],
+if applicable, is the user unit name for a session.
+
+| `systemd.t.TRANSPORT`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#_TRANSPORT=[`systemd.t.TRANSPORT`]
+is the method of entry by the journal service. This includes, `audit`, `driver`,
+`syslog`, `journal`, `stdout`, and `kernel`.
+
+| `systemd.t.UID`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#_PID=[`systemd.t.UID`]
+is the user ID for the journal entry process.
+|===
+
+[discrete]
+[[exported-fields-systemd.u]]
+=== `systemd.u` Fields
+
+`systemd.u Fields` are directly passed from clients and stored in the journal.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `systemd.u.CODE_FILE`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_FILE`]
+is the code location containing the filename of the source.
+
+| `systemd.u.CODE_FUNCTION`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_FUNCTION`]
+is the code location containing the function of the source.
+
+| `systemd.u.CODE_LINE`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#CODE_FILE=[`systemd.u.CODE_LINE`]
+is the code location containing the line number of the source.
+
+| `systemd.u.ERRNO`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#ERRNO=[`systemd.u.ERRNO`],
+if present, is the low-level error number formatted in numeric value, as a decimal string.
+
+| `systemd.u.MESSAGE_ID`
+|link:https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html#MESSAGE_ID=[`systemd.u.MESSAGE_ID`]
+is the message identifier ID for recognizing message types.
+
+| `systemd.u.RESULT`
+|For private use only.
+
+| `systemd.u.UNIT`
+|For private use only.
+|===
+
+[discrete]
+[[exported-fields-kubernetes]]
+=== Kubernetes Fields
+
+The namespace for Kubernetes-specific metadata. The `kubernetes.pod_name` is the
+name of the pod.
+
+[discrete]
+[[exported-fields-kubernetes.labels]]
+=== `kubernetes.labels` Fields
+
+Labels attached to the OpenShift object are `kubernetes.labels`. Each label name
+is a subfield of labels field. Each label name is de-dotted, meaning dots in the
+name are replaced with underscores.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `kubernetes.pod_id`
+|Kubernetes ID of the pod.
+
+| `kubernetes.namespace_name`
+|The name of the namespace in Kubernetes.
+
+| `kubernetes.namespace_id`
+|ID of the namespace in Kubernetes.
+
+| `kubernetes.host`
+|Kubernetes node name.
+
+| `kubernetes.master_url`
+|Kubernetes Master URL.
+|===
+
```

Comment:
Missed "kubernetes.container_name", please ask rmeggins to provide the detailed descripton